allofplos/plos_corpus.py

```diff
@@ -873,6 +876,55 @@ def create_local_plos_corpus(corpusdir=corpusdir, rm_metadata=True):
         os.remove(metadata_path)
 
 
+def create_test_plos_corpus(corpusdir=corpusdir):
+    """
+    Downloads a copy of 10,000 randomly selected PLOS articles by:
+    1) creating corpusdir if it doesn't exist
+    2) downloading the zip file (defaults to corpus directory)
+    3) extracting the individual XML files into the corpus directory
+    :param corpusdir: directory where the corpus is to be downloaded and extracted
+    :return: None
+    """
+    if os.path.isdir(corpusdir) is False:
+        os.mkdir(corpusdir)
+        print('Creating folder for article xml')
+    zip_path = download_file_from_google_drive(test_zip_id, local_test_zip)
+    unzip_articles(file_path=zip_path, extract_directory=corpusdir)
+
+
+def download_corpus_metadata_files(csv_abstracts=True, csv_no_abstracts=True, sqlitedb=True, destination=None):
+    """Downloads up to three files of metadata generated from the PLOS Corpus XML.
+    Includes two csvs and a sqlite database.
+    """
+    if destination is None:
+        destination = os.getcwd()
+    if csv_abstracts:
+        csv_abstracts_id = '0B_JDnoghFeEKQWlNUUJtY1pIY3c'
+        csv_abstracts_file = download_file_from_google_drive(csv_abstracts_id,
+                                                             'allofplos_metadata_test.csv',
+                                                             destination=destination)
+    if csv_no_abstracts:
+        csv_no_abstracts_id = '0B_JDnoghFeEKeEp6S0R2Sm1YcEk'
+        csv_no_abstracts_file = download_file_from_google_drive(csv_no_abstracts_id,
+                                                                'allofplos_metadata_no_abstracts_test.csv',
+                                                                destination=destination)
+    if sqlitedb:
```

Comment:
Is this a part of the current codebase? This should probably be in a separate PR